frontend/src/metabase/public/hooks/use-embed-frame-options.ts

```diff
@@ -12,14 +13,21 @@ export const useEmbedFrameOptions = ({ location }: { location: Location }) => {
     titled = DEFAULT_EMBED_DISPLAY_PARAMS.titled,
     theme = DEFAULT_EMBED_DISPLAY_PARAMS.theme,
     hide_parameters = DEFAULT_EMBED_DISPLAY_PARAMS.hideParameters,
-    hide_download_button = DEFAULT_EMBED_DISPLAY_PARAMS.hideDownloadButton,
+    hide_download_button = null,
+    downloads = DEFAULT_EMBED_DISPLAY_PARAMS.downloadsEnabled,
   } = parseHashOptions(location.hash) as DashboardUrlHashOptions;
 
+  const downloadsEnabled = PLUGIN_RESOURCE_DOWNLOADS.areDownloadsEnabled({
```

Comment:
I guess the biggest change to this PR compared to how it was before the refactors, is that I now calculate the value in this hook so that every `consumer` of this already has the value that's needed instead of having to calculate it from the two separate values.